### PR TITLE
Fix/display billable hours

### PR DESCRIPTION
**Pull request by timl07**
## Checklist
- [x] I'm using the correct PHP Version (8.1 for current, 7.4 for legacy).
- [x] I've added comments to any new methods I've created or where else relevant.
- [x] I've replaced magic method usage on DbService classes with the getInstance() static method.
- [x] I've written any documentation for new features or where else relevant in the docs [repo](https://github.com/2pisoftware/cmfive-docs).

## Changelog

refs:
issues:

## Other Information

Docs pull request:

### CODE REVIEW

File: system/modules/task/models/TaskTime.php [hunk 1]
```diff
@@ -28,5 +28,41 @@ public function getComment() {
     public function getTask() {
         return $this->getObject("Task", $this->task_id);
     }
-    
+
+    public function getTotalTimeByTimeType() {
```
Comment (careck) [inline system/modules/task/models/TaskTime.php:32]: This function should rather be in the TaskService class as it collects results from multiple tasks. Functions in DbObject derived classes should pertain only to this particular instance.

File: system/modules/task/models/TaskTime.php [hunk 2]
```diff
@@ -28,5 +28,41 @@ public function getComment() {
...
+                    $total_time_for_type += $timelog->getDuration();
+                }
+                $total_time_fmtd = floor($total_time_for_type / 3600) . gmdate(":i:s", $total_time_for_type) . '  ';
+            }
+            $time_totals_for_time_types[$timelog_type] = $total_time_fmtd;
+        }
+        return $time_totals_for_time_types;
+    }
+
+    public function getTotalTimeByBillable() {
```
Comment (careck) [inline system/modules/task/models/TaskTime.php:56]: Same as above. This function belongs to TaskService class as it collects results for multiple tasks.

**Review by careck — CHANGES_REQUESTED** [review]

File: system/modules/task/models/TaskTime.php (hunk 2 above)
Comment (timl07) [reply system/modules/task/models/TaskTime.php]: Understood, moved.

**Review by dragonflyfree — APPROVED** [review]

**Comment by chris-bateman** [comment]
@dragonflyfree can you work on these merge issues?

**Pull request closed without merging**